### ISSUE TITLE
WCAG 2.1 Understanding 2.3.1 - missing/vague dimension definitions

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#585]
In https://www.w3.org/WAI/WCAG21/Understanding/three-flashes-or-below-threshold.html it mentions

> The 1024 x 768 screen is used as the reference screen resolution for the evaluation. The 341 x 256 pixel block represents a 10 degree viewport at a typical viewing distance.

This omits / handwaves one fairly important aspect: saying that a monitor is "1024 x 768" is mostly meaningless without any physical dimensions of the monitor itself. I can set a giant cinema display to run at "1024 x 768", or a super-small LCD screen. Unless a monitor size itself is defined, or the "typical viewing distance" is explained (would this "typical distance" be the one that results in the pixels matching up with the definition for the "reference pixel"?), this is very vague.

I seem to remember discussion ages ago that this SC/Understanding's wordings came from a time when 1024x768 was the norm/top-end, and "most" monitors were generally of a certain size (14" or so?) ... but now, this feels strangely anachronistic without any anchoring/rationale behind it.

**Comment by alastc** [comment]
Hi Patrick,

So is the question about the original rational, or primarily about converting it to a more modern size?

The first might take some digging, but taking the rational of the time I assume we could convert that to a more modern definition.

E.g. the <a href="https://www.w3.org/TR/CSS21/syndata.html#x40">reference pixel</a> is about 0.0213 degrees at the intended viewing distance, so the "10 degree viewport at a typical viewing distance" is (10 / 0.0213 = 469) 469px. Hmm, might need to round that off/up.

The technique [G176](https://www.w3.org/WAI/WCAG21/Techniques/general/G176) goes into a lot more detail and should be considered at the same time, care to do a PR?

**Comment by awkawk** [comment]
The resolution defined was because of the typical monitor resolution at the time. How about we change:

The 1024 x 768 screen is used as the reference screen resolution for the evaluation. The 341 x 256 pixel block represents a 10 degree viewport at a typical viewing distance.

to

If a 1024 x 768 screen on a desktop monitor is used as the reference screen resolution for an evaluation then a 341 x 256 pixel block represents a 10 degree viewport at a typical viewing distance.

Would this help?

**Comment by alastc** [comment]
If it's updated to match CSS pixels, it appears we need to update the value as well, 341x256 is not 10 degrees according to the spec. The commenter in issue #553 came to the same conclusion.

**Comment by patrickhlauke** [comment]
Just adding here, as an aside: the 1024x768 reference is used by tools like [PEAT](http://trace.umd.edu/peat), and (at least last time I checked) what the tool does is resize your browser to that particular size, to match the reference. This is ok, except in the age of responsive/adaptive web design, where a layout may well change at different breakpoints ... and you'd really want a more abstracted way of testing this at different breakpoints, rather than only relying on testing the 1024x768 breakpoint...

**Comment by alastc** [comment]
PR added: #2127
